### PR TITLE
update dot files and fix generation

### PR DESCRIPTION
**Pull request by jeanphilippeD**
Clean up the .dot file, also indicating no changes happened with DKG implementation.

A number of .dot files were not re-generated:
 - Ensure we can re-generate all .dot files at once with
   `PARSEC_DUMP_GRAPH_SVG=0 cargo run --release --manifest-path=dot_gen/Cargo.toml -- --all -f`
 - regenerate the dot files from commit before DKG work start
 - Fix issues that change seed behaviour used to generate test .dot files.

Commit are best looked at one at a time rather than altogether.

### CODE REVIEW

File: src/dev_utils/environment.rs [hunk 1]
```diff
@@ -7,29 +7,38 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 use crate::{
-    dev_utils::network::Network, dev_utils::new_common_rng, dev_utils::RngChoice,
-    dev_utils::RngDebug, observation::ConsensusMode,
+    dev_utils::network::Network, dev_utils::network::ConsensusError, dev_utils::new_common_rng, dev_utils::RngChoice,
+    dev_utils::RngDebug, observation::ConsensusMode, dev_utils::Schedule,
```
Comment (maqi) [inline src/dev_utils/environment.rs]: I thought the auto-format will move dev_utils::Schedule before observation::ConsensusMode.

File: src/dev_utils/environment.rs [hunk 2]
```diff
@@ -7,29 +7,38 @@
...
 };
 use std::fmt;
 
 pub struct Environment {
     pub network: Network,
     pub rng: Box<RngDebug>,
+    pub rng2: Box<RngDebug>,
```
Comment (maqi) [inline src/dev_utils/environment.rs]: better have a comment explaining what the second rng is for and why it is required?

File: src/dev_utils/environment.rs (hunk 1 above)
Comment (fizyk20) [reply src/dev_utils/environment.rs]: We could collect the imports into `dev_utils::{...}`, like we do in most other places. The auto-format will then sort these.

Comment (maqi) [reply src/dev_utils/environment.rs]: yeah, that will be better

**Comment by jeanphilippeD** [comment]
Fixed review comments, force push diff show the changes

**Pull request merged**